CONTRIBUTING.md

```diff
@@ -118,7 +151,102 @@ Each subdirectory contains topical subdirectories for each chapter in that secti
 Each topical subdirectory contains:
 
   - An *index.md* file that has the metadata discussed above plus the text for the section
-  - An *assets* subdirectory where graphical files for that topic are stored
+  - An *assets* subdirectory where graphical files for that topic are stored.
 No *assets* subdirectory is present if the topic has no graphics.
 
+## Submitting new content through a pull request
+
+If you have forked and cloned the repository,
+you can modify the documentation or add new material
+by editing the markdown file(s) in the local clone of your fork
+and then submitting a _pull request (PR)_.
+
+Note that you can also modify the source using the github editor.
+This is very useful when you want to fix a typo or make a small editorial change but, if you are doing significant writing, it is generally better to work on files in your local clone.
+
+The following sequence of steps is a reasonable workflow for creating new content on your local clone and pushing it to github to be reviewed and merged.
+
+1. Execute the following command from the root folder and leave it running in a shell:
+    ```
+    hugo server -D
+    ```
+2. Create a local branch for your changes.  Be sure to base your new branch on the contents of the `master` branch unless you intend to create a derivative PR:
+   ```
+   git branch master
+   git pull
+   git checkout -b <your-new-branch>
+   ```
+3. Execute the following and check the output to ensure that your branch is set up correctly:
+   ```
+   git status
+   ```
+
+3. Do the writing you want to do in your local branch, checking the formatted version at localhost:1313.
+
+4. When you have completed the writing you want to do, close all files in your branch and run `git status` to confirm that it correctly reflects the files you have modified, added, and deleted.
+
+4. Add and commit your changes.  Here, we commit all modified files but you can specify individual files to the `git add` command.
+The `git commit -s` command commits the files and signs that you are contributing this intellectual property to the Keptn project.
+   ```
+   git add .
+   git commit -s
+   ```
+
+   Use vi commands to add a description of the PR (should be 80 characters or less) to the commit.  This will be displayed as the title of the PR in listings.
```

Comment:
```suggestion
   Use vi commands to add a description of the PR (should be 80 characters or less) to the commit. This will be displayed as the title of the PR in listings.
```